### PR TITLE
Issues #3,#6

### PR DESCRIPTION
**Pull request by harikesh409**
Made the footer and blog section responsive.

### CODE REVIEW

**Comment by sabesansathananthan** [comment]
@harikesh409  could you please resolve confllicts

**Comment by sabesansathananthan** [comment]
@harikesh409 add the screenshot of working model.

File: css/style.css [hunk 1]
```diff
@@ -2985,7 +2985,7 @@ h6 {
 ============================================================================================ */
 .footer_area {
   background: #000410;
-  height: 350px;
+  height: auto;
```
Comment (sabesansathananthan) [inline css/style.css]: no need fixed, thank you for this code chage the other part and push it

Comment (harikesh409) [reply css/style.css]: moved the code to `responsive.css` and pushed the code.

**Comment by harikesh409** [comment]
> @harikesh409 add the screenshot of working model.

medium articles
footer

**Comment by sabesansathananthan** [comment]
@harikesh409 get latest changes from master and pull it There are some conflicts

File: css/style.css [hunk 2]
```diff
@@ -1981,6 +1981,12 @@ h6 {
   transition: all 0.3s ease 0s;
 }
 
+@media(max-width:775px) {
+  .genric-btn {
+    left:80px;
+  }
+}
+
```
Comment (sabesansathananthan) [inline css/style.css]: why we need this???

File: css/responsive.css [hunk 3]
```diff
@@ -379,4 +379,19 @@
...
+    flex-wrap: wrap;
+  }
+  .medium-widget-article__item {
+    flex: 0 0 100% !important;
+  }
+}
+
+/* End Medium Widget Area css
+	============================================================================================ */
+
```
Comment (sabesansathananthan) [inline css/responsive.css]: seems good. nice work

File: css/style.css (hunk 2 above)
Comment (harikesh409) [reply css/style.css]: It's for the download cv and download resume buttons in about page.

Comment (sabesansathananthan) [reply css/style.css]: It is fixed could you please remove this part and get pull from the master branch and push it. there are some conflicts. clone my repo

**Comment by harikesh409** [comment]
> Could you please get latest changes from master and resolve conflicts

merged and resolved the conflicts

**Comment by sabesansathananthan** [comment]
@harikesh409  thank you. nice work :clap:

**Pull request merged**